app/src/main/java/com/mashup/mashup_android_6th/network/MovieRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.mashup.mashup_android_6th.network
+
+import com.mashup.mashup_android_6th.models.Response
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+object MovieRepository {
```

Comment:
이 클래스 이름보다 `MovieRepository` 보다 `MovieService` 를 추천합니다. 네트워크를 통해 (HttpRequest) 외부 자원에 접근하는 기능을 수행하기 때문에, 데이터 저장소에 접근하는 `Repository` 라는 이름보다 `Service` 이름을 쓰는게 적절하다고 생각합니다.

https://forums.asp.net/t/2068109.aspx?What+is+Repository+vs+Service+pattern
https://forums.asp.net/post/5966958.aspx
> The original definition of a service layer is one that exposes services to third parties via SOAP or HTTP over the web.

https://stackoverflow.com/questions/5049363/difference-between-repository-and-service-layer
> Repository Layer gives you additional level of abstraction over data access. Instead of writing
> Service layer exposes business logic, which uses repository. Example service could look like:

https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/building-cross-platform-applications/architecture#typical-application-layers
> Service Access Layer – Used to access services in the cloud: from complex web services (REST, JSON, WCF) to simple retrieval of data and images from remote servers. Encapsulates the networking behavior and provides a simple API to be consumed by the Application and UI layers.

https://www.quora.com/In-software-architecture-what-is-a-service-layer/answer/Scott-Nash
>  Centralizes external access to data and functions